src/components/parameters/voltage-init/voltage-init-parameters-inline.tsx

```diff
@@ -76,14 +78,26 @@ export function VoltageInitParametersInLine({
         updateVoltageInitParameters(studyUuid, {
             applyModifications: DEFAULT_GENERAL_APPLY_MODIFICATIONS,
             computationParameters: null, // null means Reset
-        }).catch((error) => {
-            snackError({
-                messageTxt: error.message,
-                headerId: 'updateVoltageInitParametersError',
+        })
+            .catch((error) => {
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'updateVoltageInitParametersError',
+                });
+            })
+            .finally(() => {
```

Comment:
Do you need to wait the end of the request to close the confirmation dialog ? I thnik you should move this outside of the finally